corehq/apps/app_manager/static/app_manager/js/details/screen.js

```diff
@@ -150,6 +153,21 @@ hqDefine("app_manager/js/details/screen", function () {
             });
         };
 
+        // Given a column model, return a boolean indicating whether the column is on an odd
+        // or an even tab, for the sake of being able to differentiate in the case tile preview
+        // which rows go with which tab.
+        self.tabPolarity = function (column) {
+            var self = this,
```

Comment:
feel free to use `const` ;)

Reply:
thanks, I haven't developed good ES6 habits yet - 904070a6db4